tests/unit/Moderation/Moderations/StickTest.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace MyBB\Core\Moderation\Moderations;
+
+use Mockery;
+use MyBB\Core\Test\ClassAssertionsTrait;
+use stdClass;
+
+class StickTest extends \PHPUnit_Framework_TestCase
+{
+    use ClassAssertionsTrait;
+
+    public function testCanBeConstructed()
+    {
+        $stick = new Stick();
+        static::assertInstanceOf('MyBB\Core\Moderation\Moderations\Stick', $stick);
```

Comment:
Again, please use `::class` so that should we decide to refactor naming/namespacing, it's a lot easier to do so.